tests/fiaas_deploy_daemon/specs/v3/data/examples/full.yml

```diff
@@ -38,6 +38,9 @@ metrics:
     path: /prometheus-metrics-here
   datadog:
     enabled: true
+    tags:
+      tag1: value1
+      tag2: value2
```

Comment:
This test data is used by both `test_v3factory.py` and `test_e2e.py`. The e2e test essentially runs a deployment using the input file (this file) into a minikube instance, and asserts that the resulting resources match manifests generally stored here: https://github.com/fiaas/fiaas-deploy-daemon/tree/master/tests/fiaas_deploy_daemon/e2e_expected. See https://github.com/fiaas/fiaas-deploy-daemon/blob/master/tests/fiaas_deploy_daemon/test_e2e.py#L150-L154 for the test definition using this file. I think the environment variables of the resulting deployment manifest will need to be updated for the test to eventually pass.

As an aside, it looks like semaphore either doesn't build branches from forks or is missing the required configuration to do so, otherwise the CI build status should show up on the PR

Reply:
Thanks! I did not run the e2e tests locally so I could not spot this. I'll get that added.